rust/Cargo.toml

```diff
@@ -0,0 +1,16 @@
+[package]
+name = "up-transport-socket-rust"
```

Comment:
suggest to remote the -rust from the name - this is a rust crate, so that's redundant information and for this reason we don't include this suffix anywhere else either.

Reply:
Removed